cli/manifest_parser/libs_by_loc_map.go

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2014-2019 Cesanta Software Limited
+// All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package manifest_parser
+
+import (
+	"sync"
+	"github.com/mongoose-os/mos/cli/build"
+)
+
+type libByLoc struct {
+	Lib *build.SWModule
+	mtx sync.Mutex
+}
+
+type libByLocMap struct {
+	m   map[string]*libByLoc
+	mtx sync.Mutex
+}
+
+func newLibByLocStr() *libByLoc {
+	return &libByLoc{
+		mtx: sync.Mutex{},
```

Comment:
it's zero value, no need to do it explicitly

Reply:
Ah sweet, I've only added those since the original string_flag_set.go and manifest_parser.go sport tons of such explicit zero-value initialisers, making me presume it was the local style.  Will gladly drop.